code/online_challenges/src/hackerrank/lonely_integer/lonely_integer.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+int main()      /*Driver Function*/
+{
+    int n,j=0,a,i;
+    scanf("%d",&n);     /*Taking input from user*/ 
+    int arr[n];
+    for(i=0;i<n;i++)
```

Comment:
Fix indentation in this file. Also use pre-increment in loops. Reformat this file with clang-format and 4 spaces (https://zed0.co.uk/clang-format-configurator/).

Reply:
@arnavb  Please check now it is correct or I have to modify it again.

Reply:
@arnavb  If everything is fine then please merge it sir.